serde-generate/src/rust.rs

```diff
@@ -243,7 +243,7 @@ where
             Option(format) => format!("Option<{}>", Self::quote_type(format, known_sizes)),
             Seq(format) => format!("Vec<{}>", Self::quote_type(format, None)),
             Map { key, value } => format!(
-                "Map<{}, {}>",
```

Comment:
What was this Map type, I can't see any type imported with that name. I suppose it's a serde specific thing?

Reply:
Not sure either. It was failing a unit test, so I fixed it. It was failing earlier too, but I guess nobody noticed.